README.md

```diff
@@ -243,14 +243,16 @@ Compatibility between this plugin and Zephyr Enterprise is:
 
 | Plugin | Zephyr Enterprise |
 |--------|-------------------|
-| v2.5   | v7.0 or later     |
+| v2.9   | v8.1 or later    |
+| v2.8   | v7.16 to v8.0.1    |
+| v2.7   | v7.0 to v7.15.1    |
 | v2.4   | v7.0 to v7.15     |
 | v2.3   | v7.0 to v7.15     |
-| v2.2   | v6.7.x to v7.15   |
+| v2.2   | v6.7.x to v7.0    |
```

Comment:
any specific reason why 7.15 has been changed to 7.0??

Reply:
This is from a previous commit 
https://github.com/SmartBear/zephyr-enterprise-test-management-plugin/commit/4f4689188ceb1ad717b4e54fb2ce7e7191bc02cb